libs/blocks/notification/notification.css

```
@@ -17,6 +17,7 @@
   --icon-size-xl: 64px;
   --pill-radius: 16px;
   --pill-shadow: 0 3px 6px #00000029;
+  --split-shadow: 0px 6px 16px 0px rgba(0, 0, 0, 0.16);
```

No need for pixel values for 0; you can use more modern color notation:
```suggestion
  --split-shadow: 0 6px 16px 0 rgb(0 0 0 / 0.16);
```

I'd also suggest we make things consistent, I see L19 has a hex declaration, `#00000029` is equivalent to `rgba(0, 0, 0, 0.16)`, we should stick to a single pattern.

---

@overmyheadandbody I have made the suggested change